bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BulkOptions.java

```diff
@@ -73,30 +73,29 @@
   public static final int BIGTABLE_BULK_THROTTLE_TARGET_MS_DEFAULT = 100;
 
   /**
-   * The maximum amount of time a row will be buffered for. By default 0: indefinitely.
+   * The maximum amount of time a row will be buffered for (default value: 1 second).
    *
    * <p>For internal use only - public for technical reasons.
    */
   @InternalApi("For internal usage only")
-  public static long BIGTABLE_BULK_AUTOFLUSH_MS_DEFAULT = 0;
+  public static long BIGTABLE_BULK_AUTOFLUSH_MS_DEFAULT = 1_000;
 
   /**
-   * Default rpc count per channel.
+   * Default rpc count per channel (default value: 1000).
    *
    * <p>For internal use only - public for technical reasons.
    */
   @InternalApi("For internal usage only")
-  public static final int BIGTABLE_MAX_INFLIGHT_RPCS_PER_CHANNEL_DEFAULT = 10;
+  public static final int BIGTABLE_MAX_INFLIGHT_RPCS_PER_CHANNEL_DEFAULT = 10 * 100;
```

Comment:
This looks wrong, I dont think we want 1000 rpcs / channel

Reply:
I am a little confused here, We have the exact figure for veneer client:
https://github.com/googleapis/java-bigtable/pull/224/files#diff-7c807fb72c1041907eaeed70e8f43243R558-R562

Should we have a little different figure for the classic client?

Reply:
I think we want 10 outstanding RPCs per channel.

1 rpc = 100 elements

so in veneer MaxOutstandingElementCount = channel_count * 100 * 10

Here, we are setting an rpc, so BIGTABLE_MAX_INFLIGHT_RPCS_PER_CHANNEL_DEFAULT = 10

Reply:
Thanks alot for the explanation.

Have addressed feedback, PTAL